templates/named.conf.erb

```diff
@@ -16,5 +16,15 @@ options  {
 include "<%= scope.lookupvar('::dns::localzonepath') %>";
 <% end -%>
 
+<%- unless scope.lookupvar('::dns::acls').empty? -%>
```

Comment:
I still think this line is redundant: if it's empty you get 0 iterations of the loop and no output, right?